tests/ha/single_pod_restart/test_server_pod_restart_api.py

```diff
@@ -921,3 +921,264 @@ def test_chunk_upload_during_server_pod_restart(self):
                                   f"Actual checksum: {download_checksum}")
         LOGGER.info("Step 9: Successfully downloaded object and verified checksum")
         LOGGER.info("ENDED: Verify chunk upload during server pod restart")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-44847")
+    def test_obj_overwrite_server_pod_restart(self):
+        """
+        Verify object overwrite before and after server pod restart
+        """
+        LOGGER.info("STARTED: Test to verify object overwrite before and after server pod restart")
+        file_size = HA_CFG["5gb_mpu_data"]["file_size"]
+
+        LOGGER.info("Creating IAM user with name %s", self.s3acc_name)
```

Comment:
Seems one IAM user is created in setup method, we can use the same as that was the step mentioned in TD it seems

Reply:
This one we will have to do in whole file.